_posts/2015-03-22-experiences-from-elasticon.markdown

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title:  "Experiences from Elastic{ON}"
+date:   2015-03-22 20:00:00"
+categories: conferences elasticon
+comments: false
```

Comment:
This won't hurt but FYI this is ignored from now on. We don't allow comments anymore.